docs/Oracle Jobs/Job Types/direct_request.md

```diff
@@ -62,7 +63,7 @@ contract MyClient is ChainlinkClient {
 }
 ```
 
-A direct request job using the following pipeline could be used to fulfill this request:
+Its called a "single word" response because aside from the requestID, the fulfill callback only receives a single word argument (the `uint256 answer`) .A direct request job using the following pipeline could be used to fulfill this request:
```

Comment:
```suggestion
This is a single word response because aside from the `requestID`, the fulfill callback receives only a single word argument in the `uint256 answer`. You can fulfill this request with a direct request job using the following pipeline:
```